docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,6 +94,13 @@ expands on what differentiates IBM, including microsimulation, from other types
 Advantages of microsimulation
 +++++++++++++++++++++++++++++
 
+Event tracking
+~~~~~~~~~~~~~~
+
+One advantage of microsimulation (as opposed to some form of cohort analysis) is the ability to record events as they occur to each simulant. Depending on the
```

Comment:
What do you mean by "cohort analysis" here? Is it the same as or different from "compartmental model" above?

Reply:
I guess I was thinking it could be a good umbrella term that would include compartmental and multiplication models, but maybe I should use the term earlier or remove it if it's confusing

Reply:
I think the term works but maybe should be intentionally introduced somewhere else above.